config/locales/views.insieme.de.yml

```
@@ -92,12 +92,12 @@ de:
 
   course_reporting:
     client_statistics:
-      disability_or_canton: Behinderung / Kanton
+      disability_or_canton: Personen mit Behinderung / Kanton
       total: Total
-      challenged: Anzahl Behinderte (Personen)
-      affiliated: Anzahl Angehörige (Personen)
-      participants: Geistig-/Lernbehinderte
-      multiple_challenged: davon Mehrfachbehinderte
+      challenged: Anzahl Personen mit Behinderung
+      affiliated: Anzahl Angehörige Personen
```

```suggestion
      affiliated: Anzahl angehörige Personen
```
"angehörige" wird in der neuen Übersetzung als Adjektiv verwendet, nicht mehr als Nomen

---

:+1: